lib/src/10_auth/auth_screen.dart

```diff
@@ -217,35 +253,18 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     };
 
     return Column(children: <Widget>[
-      TextField(
-        controller: emailController,
-        autofocus: true,
-        decoration: InputDecoration(
-          labelText: resolvePhrase(FauiPhrases.EmailTextField, "Email"),
-        ),
-        onSubmitted: (s) {
-          submit();
-        },
-      ),
-      TextField(
-        controller: passwordController,
-        obscureText: true,
-        decoration: InputDecoration(
-          labelText: resolvePhrase(FauiPhrases.PasswordTextField, "Password"),
-        ),
-        onSubmitted: (s) {
-          submit();
-        },
-      ),
+      _buildTextBox(emailController, _emailNode, _passwordNode,
+          resolvePhrase(FauiPhrases.EmailTextField, 'Email'), submit),
+      _buildTextBox(passwordController, _passwordNode, null,
+          resolvePhrase(FauiPhrases.PasswordTextField, 'Password'), submit),
       _buildError(context, _error),
-      if (_loading == true)
-        AuthProgress(
-            resolvePhrase(FauiPhrases.SigningInMessage, 'signing in...')),
-      if (_loading == false)
-        RaisedButton(
-          child: Text(resolvePhrase(FauiPhrases.SignInButton, 'Sign In')),
-          onPressed: submit,
-        ),
+      (_loading == true)
+          ? AuthProgress(
+              resolvePhrase(FauiPhrases.SigningInMessage, 'signing in...'))
```

Comment:
What's the goal of this change?

Reply:
I think I just substituted the previous if else condition by ternary operator.

Reply:
done